content/docs/themes.md

```diff
@@ -3,4 +3,4 @@ title: Themes
 weight: 110
 ---
 
-Coming soon
\ No newline at end of file
+Themes are the way to customize your background and text. You get the default. There are 20 extra themes. You can make your own theme by either by change a existing theme or make a new XML docement, whichever suits you best. Just like UGL, you can share and edit freely.
```

Comment:
For a quick-paragraph description, it's a start... But I'm going to try to rephrase things to fit more with the style of other sections in the manual, and to fix a few typos.

Feel free to comment or re-edit if you don't like the changes I made to your suggestion.